### DISCUSSION TITLE
Tell me your favorite drinks!

### DISCUSSION THREAD

**Discussion by sj-h4** [NAIST-SE/AttractingContributors#1039] [2024 Attraction Contributions]
Let me know your favorite drinks. I love coffee (especially espresso).

https://github.com/sj-h4/FavoriteDrinks

**Comment by 37integer** [comment]
milk

**Comment by natsu9731** [comment]
tea

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project 
https://github.com/polarbearpppp/AttractingContributors-Games

**Comment by thapakorn604** [comment]
Hi, I have added my drink to your topic -> https://github.com/sj-h4/FavoriteDrinks/pull/16
if you don't mind please feel free to add your song to my topic -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115
